.github/workflows/ci.yml

```diff
@@ -107,14 +95,6 @@ jobs:
           PYTHON_VERSION: ${{ matrix.python-version }}
           NOX_SESSION: ${{ matrix.nox-session }}
 
-      - if: ${{ matrix.nox-session != 'unsupported_setup_py' }}
-        name: "Upload artifact"
-        uses: "actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce"
-        with:
-          name: coverage-data
-          path: ".coverage.*"
-          if-no-files-found: error
-
```

Comment:
Only the `if: ${{ matrix.nox-session != 'unsupported_setup_py' }}`  line has to be removed 🙂